docs/source/reference/sharing.md

```diff
@@ -148,6 +148,11 @@ to know how to share your slide with GitHub pages, see the
 can take some time, and *glitches* may occur between slide transitions for this
 reason.
 
+
+### Using the Github starter template
+
+A [starter template](https://github.com/jeertmans/manim-slides-starter) is available which allows to quickly get going with a new Manim slides presentation on your Github account. The template comes ready with functionality to automate tasks using Github actions and publish to Github Pages. Please refer to the template page for usage instructions.
+
```

Comment:
Can you split lines such that they never make more than 81 characters wide?

This is more a style requirements, but makes reading the code easier :-)